ironfish-cli/src/commands/wallet/unlock.ts

```diff
@@ -59,7 +60,12 @@ export class UnlockCommand extends IronfishCommand {
         'Unlocked the wallet. Call wallet:lock or stop the node to lock the wallet again.',
       )
     } else {
-      this.log(`Unlocked the wallet for ${timeout}ms`)
+      const timeoutDuration = TimeUtils.renderSpan(timeout, {
+        hideMilliseconds: true,
+        forceSecond: true,
+        forceMinute: true
+      })
+      this.log(`Unlocked the wallet for ${timeoutDuration}.`)
```

Comment:
This seems to break with low timeout values, 

```
Unlocked the wallet for .
```

Reply:
Removed the options being passed in. Tested again with low values and confirmed to to be functional:

Reply:
Looks fixed to me. Great job, thanks for this contribution.